source/_components/climate.warmup4ie.markdown

```diff
@@ -33,11 +32,6 @@ To use this component, you have to setup an account at the warmup web site first
 </p>
 
 {% configuration %}
-name:
```

Comment:
restore this, only deleting from the example was the intention.

Reply:
Okay, Done. Hopefully correct now...

Reply:
This is good 😄